src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -463,8 +509,13 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         console.error(error);
         /* eslint-enable no-console */
 
-        // eslint-disable-next-line react/prop-types
-        this.props.onError(error, code, contextInfo);
+        const { onAnnotationsFetchError, onError } = this.props;
+
+        onError(error, code, contextInfo);
```

Comment:
Can we use the `contextInfo` object to notify the parent application of how we want it to behave? We already do something similar for other errors.